MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+include README.rst
+include LICENSE.txt
+include CHANGELOG.md
+
+recursive-include tests *
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
+
+recursive-include docs Makefile make.bat *.conf *.css *.html *.ico *.ipynb *.js *.json *.npy *.png *.py *.rst *.svg
+recursive-include examples *.csv *.gz *.ipynb *.md *.npz *.png
```

Comment:
Basically searched and included all extensions from the docs and the examples. Not sure if we want all of them or even if we want the docs and examples packaged. Feel free to let me know either way.

Reply:
Also as there are some data files listed here, it is good to be careful with packaging these in case they are big. Am happy to keep or remove these as deemed appropriate.

Reply:
We've deliberately kept the size of these small, so this should be fine.

Reply:
As long as the stuff in the doc submodules isn't pulled in, everything should be fine (i.e the reference/test data)/

Reply:
So the only submodules I see are `doc/builder` and `doc/nbpublisher`. Are we trying to exclude those?

Also I see `doc/test_data/` as an ordinary directory. Should that be removed?

Reply:
> So the only submodules I see are doc/builder and doc/nbpublisher. Are we trying to exclude those?

Yes please.

> Also I see doc/test_data/ as an ordinary directory. Should that be removed?

It should also be excluded. It probably doesn't *have* to exist (i.e could be created dynamically) but it is used - when you run notebook tests locally it fills up.

Reply:
Have added lines to prune the two submodules. These seem to work as intended.

Was unclear on `doc/test_data`. Should we keep it just in case something uses it and forgets to create it or is that a non-issue?

Reply:
The presence of the directory doesn't really matter, as you say, it might be prudent to keep it. The important thing is that nothing from inside that directory should be included.